### DISCUSSION TITLE
11/14일 금일 스마트스토어 양도양수 후 오류 문의

### DISCUSSION THREAD

**Discussion by han-mangyu** [commerce-api-naver/commerce-solutions#61] [묻고 답하기]
!

**Comment by commerce-api-naver** [comment]
---
### ⚠️ 문의 내용 내 민감정보가 포함되어있습니다. ###
작성하신 포스트 원문 내 잠재적인 문제가 될 수 있는 민감정보가 포함되어있습니다. 해당 부분을 직접 확인하시어 내용을 수정해주시기 바랍니다.
본문 내용 수정 후 우측 상단의 수정 이력("edited" 글자)으로 민감정보가 포함된 수정 이전 내용이 표시되지 않도록 수정 이력도 함께 삭제해주시기 바랍니다.
 
- 대상정보: 스크린샷 내 민감정보(이메일 등)
---

안녕하세요. 커머스API 이민석입니다.

이 공간은 커머스솔루션 개발사를 위한 공간입니다.
커머스API센터 이용 관련 문의는 [스마트스토어 고객센터 1:1문의하기](https://help.sell.smartstore.naver.com)로 문의해주시기 바랍니다.
